CHANGES.md

```diff
@@ -0,0 +1,21 @@
+## v0.1.1
+
+Conduit 0.1.1 is focused on making it easier to get started with Conduit.
+
+* Conduit can now be installed on Kubernetes clusters that use RBAC
+* The new `--skip-outbound-ports` flag for the CLI `inject` command directs Conduit to bypass proxying for specific outbound ports, making Conduit easier to use with non-gRPC or HTTP/2 protocols.
```

Comment:
This would probably read a slightly more clearly to me as:

> `conduit inject` now supports a `--skip-outbound-ports` flag that...  This makes it easier to use Conduit with non-HTTP/2 protocols.